tests/test_crawlers.py

```diff
@@ -738,7 +739,7 @@ def emulate_cwd_of_ftp(self, name):
         if name not in ["..", "folder_name", ""]:
             raise ftplib.error_perm
 
-    @mock.patch('geospaas_harvesting.crawlers.ftplib.FTP', autospec=True)
+    @mock.patch('ftplib.FTP', autospec=True)
```

Comment:
@aperrin66 
I think this is incorrect because you patch the definition not the usage! 'ftplib.FTP' is the definition of ftplib and 'geospaas_harvesting.crawlers.ftplib.FTP' is the usage of it!

Reply:
Come on, we have had this conversation before...
What you are talking about is only necessary if you use the `from ... import ...` import format.